test/quelpa-test.el

```diff
@@ -12,110 +12,124 @@
      ,@body))
 
 (quelpa-deftest quelpa-expand-recipe-test ()
```

Comment:
It's good to fix accidental indentation mistakes like this, but it would be preferable to do it in a separate PR, because it makes reviewing changes more difficult.

Reply:
Sorry, the change is for adding new test cases.

Reply:
Very well.  In the future, please, e.g. use Magit to make indentation fixes in separate commits, because it makes reviews much easier when each PR only has necessary, related changes.

Reply:
Thanks